---
title: "【11.21】例外"
date: 2023-08-07T00:13:31+09:00
tags: [""]
seq: [""]
draft: false
---

- Pythonではエラーを示すために例外を使っている
  - 関数の戻り値を特別な値にしてエラーを示す言語もある
- 特定の条件の下で失敗するコードを実行しようとするときには、適切な例外ハンドラを作って、起こる可能性のあるエラーをキャッチする必要がある
  - ある関数で例外が発生し、その関数で例外をキャッチしなｋれば、上位の関数の対応するハンドラがキャッチするまで例外は浮上していく　　
